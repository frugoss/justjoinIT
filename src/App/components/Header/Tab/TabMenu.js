import WorkOutlineOutlinedIcon from "@material-ui/icons/WorkOutlineOutlined";
import BusinessOutlinedIcon from "@material-ui/icons/BusinessOutlined";
import React from "react";
import Tab from "./Tab";
import {faNewspaper} from '@fortawesome/free-solid-svg-icons';
import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
import {faHandshake} from '@fortawesome/free-solid-svg-icons';
import TabOffers from "./TabOffers";


const tabs = [
    {
        to: '/ski-cams',
        text: 'Brand Stories',
        img: <BusinessOutlinedIcon/>,
    },
    {
        to: '/justgeekit',
        text: 'Just Geek IT',
        img: <FontAwesomeIcon icon={faNewspaper}/>,
    },
    {
        to: '/devs',
        text: 'Matchmaking',
        img: <FontAwesomeIcon icon={faHandshake}/>

    }
];


const TabMenu = ({location}) => {
    const isActive = destination => location.pathname === destination;
    return (
        <>
            <TabOffers location={location} to="/" text="Job Offers" img={<WorkOutlineOutlinedIcon/>}/>
            {tabs.map(({to, text, img}) => (
            <Tab
                img={img}
                key={to}
                to={to}
                text={text}
                isActive={isActive(to)}
            />

        ))
        }
        </>
    )
};

export default TabMenu;
