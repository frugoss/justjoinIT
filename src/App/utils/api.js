export let API_HOST = 'http://localhost:7000'
if (process.env.NODE_ENV === 'test') {
  API_HOST = 'http://localhost'
}
