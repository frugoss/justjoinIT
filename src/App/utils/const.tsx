

export interface userInterface {
    auth?: boolean,
    name?: string,
    userID?: string,
    loggedPopup?: boolean,
    offPopup?: boolean,
    createPopup?: boolean,
    addPopup?: boolean
}


export interface offerInterface {
    _id: string,
    agreements: string,
    city: string,
    company: string,
    companyIndustry: string,
    companySize: string,
    coordinates: [{ lat: number, lng: number }],
    currency: string,
    description: string,
    email: string,
    employment: string,
    experience: string,
    logo: string | undefined,
    maxSalary: number,
    minSalary: number,
    street: string,
    techStack: [{ language: string, lvl: number }],
    technology: [{ color: string, background: string, img: string, name: string }],
    title: string,
    userID: string,
    website: string,
}

export interface formInterface {
    _id?: string,
    agreements?: string,
    city?: string,
    company?: string,
    companyIndustry?: string,
    companySize?: string,
    coordinates?: [{ lat: number, lng: number }],
    currency?: string,
    description?: string,
    email?: string,
    employment?: string,
    experience?: string,
    logo?: string| undefined,
    maxSalary?: number,
    minSalary?: number,
    street?: string,
    techStack: { language?: string, lvl?: number }[],
    technology?: [{ color: string, background: string, img: string, name: string }],
    title?: string,
    userID: string,
    website?: string,
}


export interface applicationInterface {
    checkbox: boolean,
    email: string,
    file: string,
    message: string,
    name: string,
    offerID: string
}

export interface filtersInterface{
    city: string,
    language: string,
    experience: string,
    salarymax: number,
    salarymin: number,
}

export interface matchInterface{
    city: string,
    language: string,
    experience: string,
    salarymax: string,
    salarymin: string,
}

export interface formValuesCV  {
    name:string,
    email:string,
    message: string,
    checkbox: string,
    file: File,
    offerID: string
}

export interface languageInterface {
    name: string,
    to: string,
    img: string,
    color: string,
    background: string
}



export interface mobileViewModeInterface {
    view: string,
    header: boolean
}

export interface cityInterface {
    name: string,
    to: string,
    coordinates: {lat:number, lng:number}
}
