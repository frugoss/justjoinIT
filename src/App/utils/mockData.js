export const offersList = [
  {
    "_id": "5e9cfa222f9c0e0965b86034",
    "userID": "5e8cc6d853004b069507091d",
    "technology": [
      {
        "img": "data:image/png;baC",
        "color": "rgb(255, 199, 6)",
        "name": "javascript",
        "background": "rgb(255, 199, 6), rgb(255, 175, 0)"
      }
    ],
    "techStack": [
      {
        "language": "JavaScript",
        "lvl": 5
      },
      {
        "language": "ES7",
        "lvl": 3
      }
    ],
    "logo": "data:image/png;basI=",
    "coordinates": [
      {
        "lat": 54.4399555,
        "lng": 18.5440814
      }
    ],
    "city": "Sopot",
    "street": "Tuwima 20",
    "agreements": "Informujemy, że administratorem danych jest dddd z siedzibą w ___________, ul. _______________ (dalej jako \"administrator\"). Masz prawo do żądania dostępu do swoich danych osobowych, ich sprostowania, usunięcia lub ograniczenia przetwarzania, prawo do wniesienia sprzeciwu wobec przetwarzania, a także prawo do przenoszenia danych oraz wniesienia skargi do organu nadzorczego. Dane osobowe przetwarzane będą w celu realizacji procesu rekrutacji. Podanie danych w zakresie wynikającym z ustawy z dnia 26 czerwca 1974 r. Kodeks pracy jest obowiązkowe. W pozostałym zakresie podanie danych jest dobrowolne. Odmowa podania danych obowiązkowych może skutkować brakiem możliwości przeprowadzenia procesu rekrutacji. Administrator przetwarza dane obowiązkowe na podstawie ciążącego na nim obowiązku prawnego, zaś w zakresie danych dodatkowych podstawą przetwarzania jest zgoda. Dane osobowe będą przetwarzane do czasu zakończenia postępowania rekrutacyjnego i przez okres możliwości dochodzenia ewentualnych roszczeń, a w przypadku wyrażenia zgody na udział w przyszłych postępowaniach rekrutacyjnych - do czasu wycofania tej zgody. Zgoda na przetwarzanie danych osobowych może zostać wycofana w dowolnym momencie. Odbiorcą danych jest serwis Just Join IT oraz inne podmioty, którym powierzyliśmy przetwarzanie danych w związku z rekrutacją.",
    "company": "Travers",
    "companyType": "Software House",
    "companyIndustry": "Medicine",
    "currency": "PLN",
    "employment": "Mandate Contract",
    "experience": "Senior",
    "title": "Javascript developer",
    "minSalary": 15000,
    "maxSalary": 25000,
    "description": "<div><!--block--><strong>Who we are looking for<br></strong><br></div><ul><li><!--block--><strong>Senior Angular Developer (3+ experience)</strong></li><li><!--block-->Good analytical skills (problem solving)</li><li><!--block-->Good communication skills</li><li><!--block-->Enjoy helping other Engineers improve and become better</li><li><!--block-->Very good English speaking skills</li></ul><div><!--block--><strong><br>What we offer<br></strong><br></div><ul><li><!--block-->Working in a Development Team (4 team members) alongside backend team placed in London</li><li><!--block-->Lead a team of talented and collaborative engineers&nbsp;</li><li><!--block-->Help mentor and coach members of the team</li><li><!--block-->Help evolve development standards and practices</li><li><!--block-->Review Pull Requests from other developers</li><li><!--block-->Overall responsible for the technical quality of what the team is producing</li><li><!--block-->Collaborate with back-end developers, UX and UI designers to improve our products and features</li></ul><div><!--block--><strong><br>Benefits<br></strong><br></div><ul><li><!--block-->26 days of paid holidays per year (+ additionally paid bank holidays)</li><li><!--block-->30 days sick leave per year (paid 80%)</li><li><!--block--><strong>Flexible working hours</strong></li><li><!--block-->Work from home (1 day per week)</li><li><!--block--><strong>Taking part in development-related events</strong> (conferences, trainings)</li><li><!--block-->Private medical care</li><li><!--block-->MultiSport Plus Package</li></ul>",
    "companySize": "10 - 40",
    "website": "www.traversy.com"
  },
  {
    "_id": "5e9a411cfe04a70747f1a7d1",
    "userID": "5e8cc6d853004b069507091d",
    "technology": [
      {
        "img": "data:image/png;base64",
        "color": "rgb(31, 94, 182)",
        "name": "python",
        "background": "rgb(31, 94, 182), rgb(31, 123, 196)"
      }
    ],
    "techStack": [
      {
        "language": "Javascript",
        "lvl": 3
      },
      {
        "language": "C#",
        "lvl": 2
      }
    ],
    "logo": "data:image/png;",
    "companyType": "Software House",
    "companySize": "10",
    "companyIndustry": "E-commerce",
    "experience": "Mid",
    "title": "Python Developer",
    "employment": "Permanent",
    "minSalary": 3000,
    "maxSalary": 10000,
    "currency": "EUR",
    "description": "<div><!--block-->Lorem ipsum dolor sit amet, consectetur adipiscing elit. In tincidunt maximus sem vitae cursus. Integer ullamcorper quam quis molestie dictum. Nam eros augue, aliquam vitae tristique in, laoreet ac massa. Suspendisse ultricies vulputate elit, scelerisque pellentesque turpis tempor in. Donec neque est, feugiat sit amet tempor ut, gravida sit amet ligula. Donec elit ante, pharetra ac ex a, malesuada molestie velit. Vivamus cursus risus mi, quis maximus justo convallis vitae.  Nullam ultrices porttitor quam, non cursus risus convallis vel. Mauris feugiat lectus eu arcu tempor, eget hendrerit eros imperdiet. Ut vitae sodales nisi. Nam eu dolor eu libero tincidunt maximus non vitae odio. Cras neque orci, dignissim eu feugiat ac, consectetur ut nisl. Maecenas euismod sem orci. Phasellus eget nulla et sapien vulputate mollis vel tempor sem. Nulla cursus pharetra lacus, lobortis porta nulla eleifend et. Curabitur quis arcu id velit gravida euismod vitae eu turpis. In suscipit, lectus vitae maximus iaculis, mauris massa porttitor est, vel porttitor orci massa ut quam. Morbi in sapien sit amet nisl faucibus posuere.  Quisque sit amet finibus leo. Aenean placerat sit amet nulla rutrum auctor. Curabitur scelerisque eleifend purus, quis bibendum lectus venenatis vitae. Quisque pellentesque lorem ex, eu facilisis magna convallis eget. Aliquam eget fermentum augue. Ut sollicitudin dui elit, eu consectetur mi rutrum in. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Nam at massa interdum, feugiat augue ac, fringilla mauris. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus ut velit vulputate nulla luctus aliquet.  Sed ornare viverra felis vitae tincidunt. Pellentesque hendrerit massa sit amet nibh viverra lacinia. Duis vitae metus vel libero tristique euismod. Sed congue consequat purus, quis convallis ex venenatis id. Donec lacus massa, tincidunt vel erat quis, accumsan euismod erat. Curabitur leo dolor, aliquet eget fringilla a, fermentum pulvinar nisl. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos.  Proin eleifend sodales ligula in hendrerit. Nam volutpat, enim id varius bibendum, metus orci venenatis nibh, nec vestibulum ex dolor sit amet ipsum. Cras blandit enim vitae justo lobortis, vel efficitur sem luctus. Duis non dolor id erat consectetur eleifend. Nullam ultrices dictum lacus faucibus aliquet. Vestibulum eget diam cursus, fermentum turpis id, aliquet odio. Quisque vitae bibendum nibh, faucibus malesuada mi. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Mauris dictum risus vel metus convallis, vitae cursus metus faucibus. Cras placerat rhoncus enim non tempor. Integer augue odio, efficitur non diam et, luctus lobortis mi. Suspendisse laoreet vestibulum ex non pretium. Nullam rhoncus, risus nec placerat laoreet, dui neque ultrices metus, vitae congue ligula quam a magna. Nullam sodales lacus in aliquet sagittis.</div>",
    "city": "Gdańsk",
    "street": "Tuwima",
    "coordinates": [
      {
        "lat": 54.3659778,
        "lng": 18.6342856
      }
    ],
    "email": "1234@gmail.com",
    "agreements": "Informujemy, że administratorem danych jest _____xDD_________ z siedzibą w ___________, ul. _______________ (dalej jako \"administrator\"). Masz prawo do żądania dostępu do swoich danych osobowych, ich sprostowania, usunięcia lub ograniczenia przetwarzania, prawo do wniesienia sprzeciwu wobec przetwarzania, a także prawo do przenoszenia danych oraz wniesienia skargi do organu nadzorczego. Dane osobowe przetwarzane będą w celu realizacji procesu rekrutacji. Podanie danych w zakresie wynikającym z ustawy z dnia 26 czerwca 1974 r. Kodeks pracy jest obowiązkowe. W pozostałym zakresie podanie danych jest dobrowolne. Odmowa podania danych obowiązkowych może skutkować brakiem możliwości przeprowadzenia procesu rekrutacji. Administrator przetwarza dane obowiązkowe na podstawie ciążącego na nim obowiązku prawnego, zaś w zakresie danych dodatkowych podstawą przetwarzania jest zgoda. Dane osobowe będą przetwarzane do czasu zakończenia postępowania rekrutacyjnego i przez okres możliwości dochodzenia ewentualnych roszczeń, a w przypadku wyrażenia zgody na udział w przyszłych postępowaniach rekrutacyjnych - do czasu wycofania tej zgody. Zgoda na przetwarzanie danych osobowych może zostać wycofana w dowolnym momencie. Odbiorcą danych jest serwis Just Join IT oraz inne podmioty, którym powierzyliśmy przetwarzanie danych w związku z rekrutacją.",
    "website": "www.lion.com",
    "company": "Lion",
    "remote": true
  },
  {
    "_id": "5e9cf7062f9c0e0965b86029",
    "userID": "5e8cc6d853004b069507091d",
    "technology": [
      {
        "img": "data:image/png;base64",
        "color": "rgb(47, 207, 187)",
        "name": "c",
        "background": "rgb(47, 207, 187), rgb(55, 221, 156)"
      }
    ],
    "techStack": [
      {
        "language": "C",
        "lvl": 4
      },
      {
        "language": "Java",
        "lvl": 2
      },
      {
        "language": "Python",
        "lvl": 5
      }
    ],
    "logo": "data:image/jpeg;base64,",
    "currency": "USD",
    "companyType": "Startup",
    "companyIndustry": "Military",
    "email": "test",
    "employment": "B2B",
    "experience": "Mid",
    "city": "Gdynia",
    "coordinates": [
      {
        "lat": 54.51889,
        "lng": 18.53188
      }
    ],
    "street": "Hubala 10",
    "description": "<div><!--block-->The projects, you will work with, come from the most creative and innovative companies in the Silicon Valley, CA, USA and Western Europe. Take part in the things you have read about on the Internet and do the outstanding stuff here: in Poland. The real adventure is waiting for you.<br><br>About project:<br>Recognition of objects from drone photos, processing into 3D models and providing the user with platform to view and analize the results.<br><br></div><div><!--block-->Expected experience:</div><ul><li><!--block-->Very good spoken and written English</li><li><!--block-->Very good knowledge of Python or Node.JS</li><li><!--block-->Good understanding of Google Cloud, Circle CI and Kuberenetes for CI/CD</li><li><!--block-->Good knowledge of SQL and non-SQL databases</li><li><!--block-->Easy to work with</li><li><!--block-->Ambitious and excited to do new stuff</li></ul><div><!--block--><br>Appreciated knowledge:</div><ul><li><!--block-->Good practices in embedded software development</li><li><!--block-->Good understanding of microservices</li><li><!--block-->Experience in leading a small developers team</li></ul><div><!--block-->&nbsp;</div><div><!--block-->We offer:</div><ul><li><!--block-->International cutting edge projects – work with the technology leaders from CA, USA!</li><li><!--block-->Very flexible working hours, home-office during selected days possible</li><li><!--block-->Corporate-free environment, straightforward, simple, easy</li><li><!--block-->Long-term commitment from apptimia side</li></ul>",
    "title": "C Developer",
    "company": "Fine",
    "minSalary": 3000,
    "maxSalary": 5500,
    "companySize": "20 - 10 ",
    "website": "www.fine.com",
    "agreements": "Informujemy, że administratorem danych jest xD z siedzibą w ___________, ul. _______________ (dalej jako \"administrator\"). Masz prawo do żądania dostępu do swoich danych osobowych, ich sprostowania, usunięcia lub ograniczenia przetwarzania, prawo do wniesienia sprzeciwu wobec przetwarzania, a także prawo do przenoszenia danych oraz wniesienia skargi do organu nadzorczego. Dane osobowe przetwarzane będą w celu realizacji procesu rekrutacji. Podanie danych w zakresie wynikającym z ustawy z dnia 26 czerwca 1974 r. Kodeks pracy jest obowiązkowe. W pozostałym zakresie podanie danych jest dobrowolne. Odmowa podania danych obowiązkowych może skutkować brakiem możliwości przeprowadzenia procesu rekrutacji. Administrator przetwarza dane obowiązkowe na podstawie ciążącego na nim obowiązku prawnego, zaś w zakresie danych dodatkowych podstawą przetwarzania jest zgoda. Dane osobowe będą przetwarzane do czasu zakończenia postępowania rekrutacyjnego i przez okres możliwości dochodzenia ewentualnych roszczeń, a w przypadku wyrażenia zgody na udział w przyszłych postępowaniach rekrutacyjnych - do czasu wycofania tej zgody. Zgoda na przetwarzanie danych osobowych może zostać wycofana w dowolnym momencie. Odbiorcą danych jest serwis Just Join IT oraz inne podmioty, którym powierzyliśmy przetwarzanie danych w związku z rekrutacją."
  }
]

export const CV = [
  {
    "name": "Sylwester Kowalski",
    "email": "sylwesterkowalski@gmail.com",
    "message": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas sit amet convallis massa. Curabitur efficitur pretium quam ut laoreet. Ut mollis ut velit quis semper. Nam placerat ullamcorper sapien vel laoreet. Vivamus lobortis elit id massa porta condimentum. Curabitur quis nulla eu purus egestas vestibulum. Vivamus semper arcu et blandit faucibus. Fusce sit amet massa cursus, tincidunt leo ut, molestie ligula. Proin sed nisi aliquet, facilisis velit at, scelerisque nunc. Integer sapien leo, porta non pellentesque id, aliquet in magna. Vestibulum venenatis porttitor enim a consequat. Maecenas sed malesuada dolor. Vivamus facilisis laoreet sapien, eu tincidunt dolor bibendum vitae. Etiam id sodales est. Phasellus consequat quam eu nunc tempor auctor.",
    "checkbox": false,
    "offerID": "5e9cfa222f9c0e0965b86034",
    "file": "9fce5a67910bd4d71cb75a400.pdf"
  },
  {
    "name": "Kinga Sylwestrzak",
    "email": "kingasylwestrzak@gmail.com",
    "message": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas sit amet convallis massa. Curabitur efficitur pretium quam ut laoreet. Ut mollis ut velit quis semper. Nam placerat ullamcorper sapien vel laoreet. Vivamus lobortis elit id massa porta condimentum. Curabitur quis nulla eu purus egestas vestibulum. Vivamus semper arcu et blandit faucibus. Fusce sit amet massa cursus, tincidunt leo ut, molestie ligula. Proin sed nisi aliquet, facilisis velit at, scelerisque nunc. Integer sapien leo, porta non pellentesque id, aliquet in magna. Vestibulum venenatis porttitor enim a consequat. Maecenas sed malesuada dolor. Vivamus facilisis laoreet sapien, eu tincidunt dolor bibendum vitae. Etiam id sodales est. Phasellus consequat quam eu nunc tempor auctor.",
    "checkbox": true,
    "offerID": "5e9cfa222f9c0e0965b86034",
    "file": "9fce5a67910bd4d71cb75a401.pdf"
  }
]
