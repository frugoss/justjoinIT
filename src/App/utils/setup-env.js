import {server} from 'App/utils/server.js'
beforeAll(() => server.listen())
afterEach(() => server.resetHandlers())
afterAll(() => server.close())
