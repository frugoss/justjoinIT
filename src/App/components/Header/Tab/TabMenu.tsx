import WorkOutlineOutlinedIcon from "@material-ui/icons/WorkOutlineOutlined";
import BusinessOutlinedIcon from "@material-ui/icons/BusinessOutlined";
import React from "react";
import Tab from "./Tab";
import {faNewspaper} from '@fortawesome/free-solid-svg-icons';
import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
import {faHandshake} from '@fortawesome/free-solid-svg-icons';
import TabOffers from "./TabOffers";
import History from 'history';


const tabs = [
    {
        to: '/ski-cams',
        text: 'Brand Stories',
        img: <BusinessOutlinedIcon/>,
    },
    {
        to: '/justgeekit',
        text: 'Just Geek IT',
        img: <FontAwesomeIcon icon={faNewspaper}/>,
    },
    {
        to: '/devs',
        text: 'Matchmaking',
        img: <FontAwesomeIcon icon={faHandshake}/>

    }
];

type TabMenuProps = {
    location: History.Location
}

const TabMenu: React.FC<TabMenuProps> = ({location}) => {
    const isActive = (destination: string) => location.pathname === destination;
    return (
        <>
            <TabOffers location={location} to="/" text="Job Offers" img={<WorkOutlineOutlinedIcon/>}/>
            {tabs.map(({to, text, img}) => (
            <Tab
                img={img}
                key={to}
                to={to}
                text={text}
                isActive={isActive(to)}
            />

        ))
        }
        </>
    )
};

export default TabMenu;
