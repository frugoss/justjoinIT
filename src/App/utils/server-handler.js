import {rest} from 'msw'
import {offersList, CV} from 'App/utils/mockData.js'

const handlers = [
  rest.post('/devs', async (req, res, ctx) => {
    return res(ctx.text("4325325325235"))
  }),
  rest.post('/devs/signup', async (req, res, ctx) => {
    return res(ctx.text("4325325325235"))
  }),
  rest.get('/', async (req, res, ctx) => {
    return res(ctx.json(offersList))
  }),
  rest.get('/cv/get', async (req, res, ctx) => {
    return res(ctx.json(CV))
  })
]

export {handlers}
