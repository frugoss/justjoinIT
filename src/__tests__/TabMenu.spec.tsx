import React from 'react'
import TabMenu from "../App/components/Header/Tab/TabMenu";
import {renderWithRouter} from "../App/utils/tests";


test('Snapshot',() => {
    const {container} = renderWithRouter(<TabMenu location={{pathname:"", search:"", state: "", hash: ""}}/>)
    expect(container).toMatchSnapshot()
})
