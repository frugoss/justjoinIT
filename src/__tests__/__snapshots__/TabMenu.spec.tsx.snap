// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot 1`] = `
<div>
  <a
    class="linkStyle"
    href="/"
  >
    <button
      class="MuiButtonBase-root MuiButton-root MuiButton-text tab makeStyles-root-1 "
      tabindex="0"
      type="button"
    >
      <span
        class="MuiButton-label"
      >
        <span
          class="MuiButton-startIcon MuiButton-iconSizeMedium"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M14 6V4h-4v2h4zM4 8v11h16V8H4zm16-2c1.11 0 2 .89 2 2v11c0 1.11-.89 2-2 2H4c-1.11 0-2-.89-2-2l.01-11c0-1.11.88-2 1.99-2h4V4c0-1.11.89-2 2-2h4c1.11 0 2 .89 2 2v2h4z"
            />
          </svg>
        </span>
        Job Offers
      </span>
      <span
        class="MuiTouchRipple-root"
      />
    </button>
  </a>
  <a
    class="linkStyle"
    href="/ski-cams"
  >
    <button
      class="MuiButtonBase-root MuiButton-root MuiButton-text tab makeStyles-root-2 "
      tabindex="0"
      type="button"
    >
      <span
        class="MuiButton-label"
      >
        <span
          class="MuiButton-startIcon MuiButton-iconSizeMedium"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M12 7V3H2v18h20V7H12zM6 19H4v-2h2v2zm0-4H4v-2h2v2zm0-4H4V9h2v2zm0-4H4V5h2v2zm4 12H8v-2h2v2zm0-4H8v-2h2v2zm0-4H8V9h2v2zm0-4H8V5h2v2zm10 12h-8v-2h2v-2h-2v-2h2v-2h-2V9h8v10zm-2-8h-2v2h2v-2zm0 4h-2v2h2v-2z"
            />
          </svg>
        </span>
        Brand Stories
      </span>
      <span
        class="MuiTouchRipple-root"
      />
    </button>
  </a>
  <a
    class="linkStyle"
    href="/justgeekit"
  >
    <button
      class="MuiButtonBase-root MuiButton-root MuiButton-text tab makeStyles-root-2 "
      tabindex="0"
      type="button"
    >
      <span
        class="MuiButton-label"
      >
        <span
          class="MuiButton-startIcon MuiButton-iconSizeMedium"
        >
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-newspaper fa-w-18 "
            data-icon="newspaper"
            data-prefix="fas"
            focusable="false"
            role="img"
            viewBox="0 0 576 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M552 64H88c-13.255 0-24 10.745-24 24v8H24c-13.255 0-24 10.745-24 24v272c0 30.928 25.072 56 56 56h472c26.51 0 48-21.49 48-48V88c0-13.255-10.745-24-24-24zM56 400a8 8 0 0 1-8-8V144h16v248a8 8 0 0 1-8 8zm236-16H140c-6.627 0-12-5.373-12-12v-8c0-6.627 5.373-12 12-12h152c6.627 0 12 5.373 12 12v8c0 6.627-5.373 12-12 12zm208 0H348c-6.627 0-12-5.373-12-12v-8c0-6.627 5.373-12 12-12h152c6.627 0 12 5.373 12 12v8c0 6.627-5.373 12-12 12zm-208-96H140c-6.627 0-12-5.373-12-12v-8c0-6.627 5.373-12 12-12h152c6.627 0 12 5.373 12 12v8c0 6.627-5.373 12-12 12zm208 0H348c-6.627 0-12-5.373-12-12v-8c0-6.627 5.373-12 12-12h152c6.627 0 12 5.373 12 12v8c0 6.627-5.373 12-12 12zm0-96H140c-6.627 0-12-5.373-12-12v-40c0-6.627 5.373-12 12-12h360c6.627 0 12 5.373 12 12v40c0 6.627-5.373 12-12 12z"
              fill="currentColor"
            />
          </svg>
        </span>
        Just Geek IT
      </span>
      <span
        class="MuiTouchRipple-root"
      />
    </button>
  </a>
  <a
    class="linkStyle"
    href="/devs"
  >
    <button
      class="MuiButtonBase-root MuiButton-root MuiButton-text tab makeStyles-root-2 "
      tabindex="0"
      type="button"
    >
      <span
        class="MuiButton-label"
      >
        <span
          class="MuiButton-startIcon MuiButton-iconSizeMedium"
        >
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-handshake fa-w-20 "
            data-icon="handshake"
            data-prefix="fas"
            focusable="false"
            role="img"
            viewBox="0 0 640 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M434.7 64h-85.9c-8 0-15.7 3-21.6 8.4l-98.3 90c-.1.1-.2.3-.3.4-16.6 15.6-16.3 40.5-2.1 56 12.7 13.9 39.4 17.6 56.1 2.7.1-.1.3-.1.4-.2l79.9-73.2c6.5-5.9 16.7-5.5 22.6 1 6 6.5 5.5 16.6-1 22.6l-26.1 23.9L504 313.8c2.9 2.4 5.5 5 7.9 7.7V128l-54.6-54.6c-5.9-6-14.1-9.4-22.6-9.4zM544 128.2v223.9c0 17.7 14.3 32 32 32h64V128.2h-96zm48 223.9c-8.8 0-16-7.2-16-16s7.2-16 16-16 16 7.2 16 16-7.2 16-16 16zM0 384h64c17.7 0 32-14.3 32-32V128.2H0V384zm48-63.9c8.8 0 16 7.2 16 16s-7.2 16-16 16-16-7.2-16-16c0-8.9 7.2-16 16-16zm435.9 18.6L334.6 217.5l-30 27.5c-29.7 27.1-75.2 24.5-101.7-4.4-26.9-29.4-24.8-74.9 4.4-101.7L289.1 64h-83.8c-8.5 0-16.6 3.4-22.6 9.4L128 128v223.9h18.3l90.5 81.9c27.4 22.3 67.7 18.1 90-9.3l.2-.2 17.9 15.5c15.9 13 39.4 10.5 52.3-5.4l31.4-38.6 5.4 4.4c13.7 11.1 33.9 9.1 45-4.7l9.5-11.7c11.2-13.8 9.1-33.9-4.6-45.1z"
              fill="currentColor"
            />
          </svg>
        </span>
        Matchmaking
      </span>
      <span
        class="MuiTouchRipple-root"
      />
    </button>
  </a>
</div>
`;
